x-pack/plugins/cases/public/components/all_cases/all_cases_list.tsx

```diff
@@ -70,7 +70,7 @@ export const AllCasesList = React.memo<AllCasesListProps>(
     const firstAvailableStatus = head(difference(caseStatuses, hiddenStatuses));
     const initialFilterOptions = {
       ...(!isEmpty(hiddenStatuses) && firstAvailableStatus && { status: firstAvailableStatus }),
-      owner: hasOwner ? owner : availableSolutions,
+      owner: hasOwner ? owner : [],
```

Comment:
I changed it because having `availableSolutions` in the initial filters will select all owners in the filter. This seemed to me not a good UX because if the user wants to filter only by a specific owner he/she has to deselect all others. This follows the same behavior as tags, reporters, etc.